src/styles/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 198px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+}
+
+.card__content {
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  margin: 0 auto 40px;
```

Comment:
this rule looks unnecessary for this element because you alined it with margin: auto;

Reply:
as I understand this margin should be here
40px - bottom margin and auto is for centering picture